src/component/FeatureRenderer.js

```diff
@@ -62,36 +62,36 @@
  *
  * @class GeoExt.component.FeatureRenderer
  */
-Ext.define('GeoExt.component.FeatureRenderer', {
-    extend: 'Ext.Component',
-    alias: 'widget.gx_renderer',
+Ext.define("GeoExt.component.FeatureRenderer", {
```

Comment:
> This would change my whole life ... :laughing:

I wouldn't want to do that. 😢 

> If we want consistency, maybe the single quote would be less evasive?

I'm totally fine with that. The only arguments that I would bring up against that is template strings and the fact that we will need to escape stuff like `'it\'s'`in the english language.